data.sql

```diff
@@ -22,4 +22,12 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
 
 -- species
-INSERT INTO species (name) VALUES ('pokemon'), ('Digimon');
\ No newline at end of file
+INSERT INTO species (name) VALUES ('pokemon'), ('Digimon');
+
+-- project 4
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2020-04-23'), ('Maisy Smith', 26, '2019-01-17'), (' Stephanie Mendez', 64, '1981-05-04'), ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations(vets_id, species_id) VALUES(1, 1), (3, 2), (3,2), (4,2);
```

Comment:
- [ ] I believe there is a little mistake in this line related to vets id. I believe that the second value should be `(3, 1)` so this line would be `
INSERT INTO specializations(vets_id, species_id) VALUES(1, 1), (3, 1), (3,2), (4,2);`